scopes/ui-foundation/use-box/dropdown/dropdown.composition.tsx

```diff
@@ -38,15 +48,16 @@ UseBoxExample.canvas = {
 export const UseBoxScopeExample = () => {
   return (
     <ThemeCompositions>
-      <div style={styles}>
-        <UseBoxDropdown position="bottom-end" Menu={() => <ScopeMenu scopeName="@teambit.design" />}></UseBoxDropdown>
+      <div>
+        <UseBoxDropdown position="bottom-end" Menu={<ScopeMenu scopeName="@teambit.design" />}></UseBoxDropdown>
       </div>
     </ThemeCompositions>
   );
 };
 
 UseBoxScopeExample.canvas = {
```

Comment:
canvas is for overview page only, better use style={} directly on the themeCompositions